raiden_contracts/tests/test_contracts_compilation.py

```diff
@@ -113,11 +113,14 @@ def test_current_development_version():
         assert source_path.exists()
     assert contracts_precompiled_path().exists()
 
-    # TODO: uncomment after testnet deployment
-    # see https://github.com/raiden-network/raiden-contracts/issues/444)
-    # assert contracts_deployed_path(NETWORKNAME_TO_ID['rinkeby']).exists()
-    # assert contracts_deployed_path(NETWORKNAME_TO_ID['ropsten']).exists()
-    # assert contracts_deployed_path(NETWORKNAME_TO_ID['kovan']).exists()
+    # deployment files exist
+    assert contracts_deployed_path(NETWORKNAME_TO_ID['rinkeby']).exists()
+    assert contracts_deployed_path(NETWORKNAME_TO_ID['ropsten']).exists()
+    assert contracts_deployed_path(NETWORKNAME_TO_ID['kovan']).exists()
+    # deployment files for service contracts also exist
+    assert contracts_deployed_path(NETWORKNAME_TO_ID['rinkeby'], None, True).exists()
+    assert contracts_deployed_path(NETWORKNAME_TO_ID['ropsten'], None, True).exists()
+    assert contracts_deployed_path(NETWORKNAME_TO_ID['kovan'], None, True).exists()
```

Comment:
I have started to use `services=True`, because it is hard for other people to know what `True` stands for.

Reply:
Will change so.  I was wondering what to do.